app/Resources/views/admin/blog/new.html.twig

```diff
@@ -1,5 +1,16 @@
 {% extends 'admin/layout.html.twig' %}
 
+{% block javascripts %}
+    {{ parent() }}
+    <script>
+        $('a[data-toggle="tab"]').on('click', function (e) {
+            if ('#previewId' == $(e.target).attr('href')) {
+                SymfonyDemo.preview('#post_content', '#previewId', '{{ path('preview') }}');
+            }
+        });
+    </script>
```

Comment:
You can create a template file to do that and avoid duplicated code in `edit.html.twig`.